LamestWebserver/LamestWebserver/HttpRequest.cs

```diff
@@ -449,8 +457,13 @@ private static HttpRequest GetCookiesAndModified(HttpRequest packet, string[] li
                 {
                     packet.IsWebsocketUpgradeRequest = true;
                 }
+                else if(packet.Range == null && linput[i].Contains("Range: bytes="))
+                {
+                   string[] beginEndPair = linput[i].Substring(("Range: bytes=").Length).Split('-');
+                   packet.Range = new Tuple<int, int>(int.Parse(beginEndPair[0]), int.Parse(beginEndPair[1])); 
```

Comment:
`int.Parse` could throw an Exception. `int.TryParse` looks not as sleek, but will just return false if it fails. Also [0] and [1] might throw an `IndexOutOfRangeException`.